examples/spark2/src/main/java/org/apache/carbondata/examples/sdk/SDKS3ReadExample.java

```diff
@@ -35,60 +39,69 @@
  * Example for testing carbonReader on S3
  */
 public class SDKS3ReadExample {
-    public static void main(String[] args) throws Exception {
-        Logger logger = LogServiceFactory.getLogService(SDKS3ReadExample.class.getName());
-        if (args == null || args.length < 3) {
-            logger.error("Usage: java CarbonS3Example: <access-key> <secret-key>"
-                + "<s3-endpoint> [table-path-on-s3]");
-            System.exit(0);
-        }
+  public static void main(String[] args) throws Exception {
+    Logger logger = LogServiceFactory.getLogService(SDKS3ReadExample.class.getName());
+    if (args == null || args.length < 3) {
+      logger.error("Usage: java CarbonS3Example: <access-key> <secret-key>"
+          + "<s3-endpoint> [table-path-on-s3]");
+      System.exit(0);
+    }
+
+    String path = "s3a://sdk/WriterOutput/carbondata5";
+    if (args.length > 3) {
+      path = args[3];
+    }
+
+    // 1. read with file list
+    Configuration conf = new Configuration();
 
-        String path = "s3a://sdk/WriterOutput/carbondata5";
-        if (args.length > 3) {
-            path=args[3];
-        }
+    conf.set(ACCESS_KEY, args[0]);
+    conf.set(SECRET_KEY, args[1]);
+    conf.set(ENDPOINT, args[2]);
+    List fileLists = listFiles(path, CarbonTablePath.CARBON_DATA_EXT, conf);
 
-        // Read data
-        EqualToExpression equalToExpression = new EqualToExpression(
-            new ColumnExpression("name", DataTypes.STRING),
-            new LiteralExpression("robot1", DataTypes.STRING));
+    // Read data
```

Comment:
why whole file is modified ? just few section can be modified. please check formatting.

Reply:
code style is not correct.